src/main.rs

```diff
@@ -115,15 +115,17 @@ impl FromStr for Show {
 #[derive(Clone, Copy)]
 pub enum DurationStyle {
     HMS,
+    MS,
     S,
 }
 impl FromStr for DurationStyle {
     type Err = String;
     fn from_str(s: &str) -> Result<Self, Self::Err> {
         match s {
             "hms" => Ok(DurationStyle::HMS),
+            "ms" => Ok(DurationStyle::MS),
             "s" => Ok(DurationStyle::S),
-            _ => Err("Valid values are 'hms', 's'.".into()),
+            _ => Err("Valid values are 'hms', 'ms', 's'.".into()),
```

Comment:
Bit of bikeshedding: "ms" reads like "milliseconds" to me. What about `hms_fixed` or similar ?